modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/architect/filter/FilterExpressionVisitor.java

```diff
@@ -65,20 +63,6 @@ public Object visitMember(Member member) {
 		return String.valueOf(resourcePath.get(0));
 	}
 
-	private static String _removeLeadingAndTrailingSingleQuotes(String s) {
-		if (Validator.isNull(s) || (s.length() == 1)) {
-			return s;
-		}
-
-		if ((s.charAt(0) == CharPool.APOSTROPHE) &&
```

Comment:
We can not use StringUtil.unquote method since it removes single quotes (') and double quotes (''), and we just need to remove the single quote (').